modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateLocalServiceTest.java

```diff
@@ -77,40 +105,43 @@ public void testAddNotificationTemplate() throws Exception {
 
 		Assert.assertNotNull(notificationRecipient);
 
-		NotificationRecipientSetting notificationRecipientSetting =
-			_notificationRecipientSettingLocalService.
-				getNotificationRecipientSetting(
-					notificationRecipient.getNotificationRecipientId(), "from");
+		_assertNotificationRecipientSetting(
+			"from", notificationRecipient.getNotificationRecipientId());
+		_assertNotificationRecipientSetting(
+			"fromName", notificationRecipient.getNotificationRecipientId());
+		_assertNotificationRecipientSetting(
+			"to", notificationRecipient.getNotificationRecipientId());
 
-		Assert.assertEquals("from", notificationRecipientSetting.getName());
-		Assert.assertEquals(
-			externalReferenceCode,
-			notificationRecipientSetting.getValue(LocaleUtil.getDefault()));
+		_notificationTemplateLocalService.deleteNotificationTemplate(
+			notificationTemplate);
+	}
 
-		notificationRecipientSetting =
+	private void _assertNotificationRecipientSetting(
+			String name, long notificationRecipientId)
+		throws Exception {
+
+		NotificationRecipientSetting notificationRecipientSetting =
 			_notificationRecipientSettingLocalService.
-				getNotificationRecipientSetting(
-					notificationRecipient.getNotificationRecipientId(),
-					"fromName");
+				getNotificationRecipientSetting(notificationRecipientId, name);
 
-		Assert.assertEquals("fromName", notificationRecipientSetting.getName());
+		Assert.assertEquals(name, notificationRecipientSetting.getName());
 		Assert.assertEquals(
-			externalReferenceCode,
+			_externalReferenceCode,
 			notificationRecipientSetting.getValue(LocaleUtil.getDefault()));
+	}
 
-		notificationRecipientSetting =
-			_notificationRecipientSettingLocalService.
-				getNotificationRecipientSetting(
-					notificationRecipient.getNotificationRecipientId(), "to");
+	@Inject
+	private CounterLocalService _counterLocalService;
```

Comment:
Since you refactored this class, I believe these variables are not being used anymore: _counterLocalService, _notificationQueueEntryLocalService, _notificationRecipientLocalService;

Could you please double-check? If positive, could you please remove it?